doc/classes/@GlobalScope.xml

```diff
@@ -525,6 +525,27 @@
 				Returns [code]true[/code] if [param x] is a NaN ("Not a Number" or invalid) value.
 			</description>
 		</method>
+		<method name="is_same">
+			<return type="bool" />
+			<param index="0" name="a" type="Variant" />
+			<param index="1" name="b" type="Variant" />
+			<description>
+				Returns [code]true[/code] if [param a] is of the same type and value of [param b] for value types. For reference types, returns [code]true[/code] if the reference is the same.
+				[codeblock]
+				# Vector2 is a value type
+				var vec2_one = Vector2(0, 0)
+				var vec2_two = Vector2(0, 0)
+				print(is_same(vec2_one, vec2_one))  # returns `true`
+				print(is_same(vec2_one, vec2_two))  # returns `true`
```

Comment:
Should the comment be "# returns `false`" here or did I misunderstand the code?

Reply:
Value types compared by value. `is_same(Vector2(0, 0), Vector2(0, 0))` returns true.

Reply:
I'll add a comment about this.

Reply:
Maybe they should be named `vec2_a` and `vec2_b` because there is `Vector2.ONE` so `vec2_one` and `vec2_two` might make you think that they have those numbers in them. Not important though.